src/modules/elements.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- Hey, as this is considered a bad practise, let's try to write code that conforms to the linter. please, kindly remove this eslint disable and try fixing the error.